src/components/admin/form-components/validated-fields.jsx

```diff
@@ -17,6 +17,15 @@ export const cannotNull = value => {
   return null;
 };
 
+export const hasVarCharMax = value => {
```

Comment:
I see what you're trying to do with the VarChar, but actually VarChar max is 255, and it's just Ghost that arbitrarily chose that slug and title should have max length 150, which is for example a different length than the teaser (that field is actually 255, though we of course as you know restrict it further than that). 

Would it be possible within the paradigm to implement this as a function factory that takes a length argument that could be given as a prop to BoundedRequiredTextField and still work? I guess it would involve creating the child component on `componentWillMount` and `componentWillReceiveProps` and saving it in `this.child` or something along those lines...

A simpler way is to maybe just call this `max150Chararacters` and call the field `Max150RequiredTextField` though it becomes very verbose... I just feel like if I read `BoundedRequiredTextField` I feel like I should be able to tell it how bounded I want it, no?

Reply:
I agree that this is confusingly named, but should note that when I said VarChar max I wasn't referring to the max as set by the number of bytes for this VarChar, but the literal max argument for VarChar for these fields(see below). I understand what you mean about 255 being the max for 1 byte, but both are still VarChar maxes in the SQL sense not the byte sense.

![screenshot 2018-03-30 at 10 40 45 pm](https://user-images.githubusercontent.com/9847335/38149472-a91e19c2-346b-11e8-8167-13791539dff0.png)

Again, Bound in this sense has nothing to do with the length, though I see how that confusion arose since these were in the same commit. For now, I will change the name of this one to `hasGhostVarCharMax` and `GhostRequiredTextField`. 

I think the function factory is over-engineering. Using validators for max length is only intended for  DB max limits that are unlikely to be hit. Presently,  the only options for that would be 150 or 255 (which isn't 3 times as your favorite rule for over-engineering would note 😉). 

If the max length is more variable than that it's likely the engineer will want to explain to the users why it is capped (e.g. non-database reason) and `MaxLenTextField` should be used which handles variable max length and fully prevents the user from entering too long a string.

Reply:
I never knew of the describe command, that's nice! I always used `show create table`!

I'll reply in the general PR